pom.xml

```diff
@@ -1584,28 +1586,22 @@
       </activation>
       <properties>
         <!-- There is no public ECJ release (yet) which supports Java 9, so fallback to the native compiler -->
-        <!-- TODO remove the line below (and fix the surefire+failsafe config) after resolving https://issues.jboss.org/browse/DROOLS-1169 -->
-        <drools.dialect.java.compiler>NATIVE</drools.dialect.java.compiler>
```

Comment:
Are you sure this line can be removed?
It's for running the tests, not compation and the related ECJ issue DROOLS-1169 isn't resolved yet.

Reply:
Yes, I just tried it locally and all works as expected with ECJ 4.4.2.